modtools/deep-embark.lua

```diff
@@ -115,9 +122,46 @@ function getValidEmbarkTiles(block)
   return validTiles
 end
 
-function reveal(pos, recenter)
--- creates an unbound glowing barrier at the target location to trigger proper tile revelation when it disappears (fortress mode only)
--- if recenter is true, shifts the view to the target location
+function blockGlowingBarrierAnnouncements(recenter)
+--  temporarily disables the "glowing barrier has disappeared" announcement
+--  announcement settings are restored after 1 tick
+--  setting recenter to true enables recentering of game view to the announcement position
+  local announcementFlags = df.global.d_init.announcements.flags.ENDGAME_EVENT_1 -- glowing barrier disappearance announcement
+  local oldFlags = df.global.d_init.announcements.flags.ENDGAME_EVENT_1:new()
+  oldFlags:assign(announcementFlags) -- backup announcement settings
```

Comment:
I think :new() creates a copy, so you don't need the :assign() call (but I have yet to actually test this)

Reply:
Looks like this is correct; I'll remove it.